Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -290,6 +291,11 @@ internal static SFSessionProperties ParseConnectionString(string connectionStrin
             return properties;
         }
 
+        internal static string ResolveConnectionAreaMessage(string host) =>
+            host.EndsWith(".snowflakecomputing.cn", StringComparison.InvariantCultureIgnoreCase)
+                ? "Connecting to CHINA Snowflake domain"
+                : "Connecting to GLOBAL Snowflake domain";

```

Comment:
is Global a good name? we have addresses in Snowflake with `.global.snowflakecomputing.com`